### ISSUE TITLE
build_network.py: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by abhinab303** [ytc272098215/HGAT#13]
on this line:
`entityList = json.loads(entityList)` 
when `entityList` input string equals to `null`(which was produced for my dataset by tagMe.py), `entityList` would be of type `None` and it will cause error while iterating on upcoming lines. 

For now I assigned `entityList` to an empty list when it is `None`.

**Comment by 1yangchunlian** [comment]
Hello, I also met this problem, have you solved it?

**Comment by abhinab303** [comment]
> Hello, I also met this problem, have you solved it?

yes. I already mentioned the fix. 
`entity_list = [] if entity_list is None`

**Comment by Jungle-liu** [comment]
Hello, can you run build_ Network, what is the distribution of training set and test set? My training set is the number of tags * 2, but my number of tags is only 2, making my training set 4.
trainnum 4
4 4 79993
train: 4
vali: 4
test: 79993
AllTexts: 80001
text-ent: : 48366it [00:01, 28899.68it/s]
